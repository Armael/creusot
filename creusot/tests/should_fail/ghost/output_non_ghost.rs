extern crate creusot_contracts;
use creusot_contracts::*;

pub fn output_non_ghost() {
    let _ = ghost!(1);
}
