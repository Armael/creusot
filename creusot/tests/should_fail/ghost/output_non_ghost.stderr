error: the return type of a ghost! block should be either () or a GhostBox
 --> output_non_ghost.rs:5:13
  |
5 |     let _ = ghost!(1);
  |             ^^^^^^^^^
  |
note: return type is i32
 --> output_non_ghost.rs:5:13
  |
5 |     let _ = ghost!(1);
  |             ^^^^^^^^^
  = note: this error originates in the macro `ghost` (in Nightly builds, run with -Z macro-backtrace for more info)

error: aborting due to 1 previous error

